source/lib-intro.tex

```diff
@@ -3022,106 +3022,128 @@
 
 \pnum
 In namespace \tcode{std}, the following names are reserved for previous standardization:
```

Comment:
Since the tables are floats, we probably want to refer to them by number/reference rather than "the following"?

Reply:
Good catch. That's even an ISO requirement.

Also, the table labels should start with the section label "zombie.names", always.  So, it's "zombie.names.om" for object macros, maybe. And "zombie.names.fm" for function macros.